runtime/vm/CRIUHelpers.cpp

```diff
@@ -1582,6 +1617,9 @@ criuCheckpointJVMImpl(JNIEnv *env,
 		U_32 intGhostFileLimit = 0;
 		IDATA criuDumpReturnCode = 0;
 		bool restoreFailure = false;
+		size_t rssBefore = 0;
+		size_t rssAfter = 0;
+		OMR::RSSReport *rssReport = OMR::RSSReport::instance();
```

Comment:
you wont be able to use this here, you'll need to add it to jitconfig like Gita did with `jitAddNewLowToHighRSSRegion`

Reply:
@tajila @gita-omr how can I adjust the change at https://github.com/eclipse-openj9/openj9/pull/19829/commits/fbdc2ca53427c8c126d18cbe69f92fcb22eaca0d to avoid the build from getting stuck during `make all`?

```
// with the change
make all
...
100 warnings
```

```
// without the change
make all
...
100 warnings
Stopping javac server
Finished building target 'all' in configuration 'linux-x86_64-server-release'

Reply:
It seems like `jdk21` and not `jdknext` can be utilized to avoid the build from getting stuck during `make all`.